CMakeLists.txt

```
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.12)
+cmake_minimum_required(VERSION 3.20)
```

```suggestion
cmake_minimum_required(VERSION 3.12)
```
This change is not required